### PR TITLE
Add functionality for finding and removing `core.*` files.

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: scripts/_Team_Defines.bash [hunk 1]
```diff
@@ -81,6 +81,31 @@ alias ros2_control_setup-controller-package=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/ro
...
+
+# Temporary files cleaning
+function rtw-find-and-remove-core-files () {
+  if [ -z "$1" ]; then
+    search_dir="${HOME}"
+  else
+    search_dir=$1
+  fi
+
+  find "${search_dir}" -type f -regex ".*core\.[0-9][0-9][0-9][0-9][0-9][0-9]"
```
Comment (mamueluth) [inline scripts/_Team_Defines.bash:98]: find "${search_dir}" -type f -regex ".*core\.[0-9]+"

core files can have more or less then 6 digits. If you want to have with a minimum number min_number then
find "${search_dir}" -type f -regex ".*core\.[0-9]\{min_number,\}"

File: scripts/_Team_Defines.bash [hunk 2]
```diff
@@ -81,6 +81,31 @@ alias ros2_control_setup-controller-package=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/ro
...
+
+  user_decision "Do you want to delete those files?" user_answer
+  if [[ " ${positive_answers[*]} " =~ " ${user_answer} " ]]; then
+    find "${search_dir}" -type f -regex ".*core\.[0-9][0-9][0-9][0-9][0-9][0-9]" -delete
```
Comment (mamueluth) [inline scripts/_Team_Defines.bash:102]: same as befor:
`find "${search_dir}" -type f -regex ".*core\.[0-9]+"`

core files can have more or less then 6 digits. If you want to have with a minimum number min_number then
`find "${search_dir}" -type f -regex ".*core\.[0-9]\{min_number,\}"`

Comment (destogl) [reply scripts/_Team_Defines.bash:102]: The pattern with `[0-9]\{number\}` didn't work for me. Does this work for you? If yes please do a suggestion so I can just add it to PR.

Comment (mamueluth) [reply scripts/_Team_Defines.bash:102]: works both for me if you don't want to supply max then i would prefer the first one. this is more understandable in my opition.

Comment (mamueluth) [inline scripts/_Team_Defines.bash:102]:
```suggestion
`find "${search_dir}" -type f -regex ".*core\.[0-9]+"`
```

File: scripts/_Team_Defines.bash (hunk 1 above)
Comment (mamueluth) [inline scripts/_Team_Defines.bash:98]:
```suggestion
`find "${search_dir}" -type f -regex ".*core\.[0-9]+"`
```

File: scripts/_Team_Defines.bash [hunk 3]
```diff
@@ -81,6 +81,31 @@ alias ros2_control_setup-controller-package=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/ro
...
+    search_dir="${HOME}"
+  else
+    search_dir=$1
+  fi
+
+find "${search_dir}" -type f -regex ".*core\.[0-9]+"
+
+  user_decision "Do you want to delete those files?" user_answer
+  if [[ " ${positive_answers[*]} " =~ " ${user_answer} " ]]; then
+find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```
Comment (destogl) [inline scripts/_Team_Defines.bash:102]:
```suggestion
  find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```

File: scripts/_Team_Defines.bash [hunk 4]
```diff
@@ -81,6 +81,31 @@ alias ros2_control_setup-controller-package=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/ro
...
+
+# Temporary files cleaning
+function rtw-find-and-remove-core-files () {
+  if [ -z "$1" ]; then
+    search_dir="${HOME}"
+  else
+    search_dir=$1
+  fi
+
+find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```
Comment (destogl) [inline scripts/_Team_Defines.bash:98]:
```suggestion
  find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```

File: scripts/_Team_Defines.bash [hunk 5]
```diff
@@ -81,6 +81,31 @@ alias ros2_control_setup-controller-package=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/ro
...
+    search_dir="${HOME}"
+  else
+    search_dir=$1
+  fi
+
+  find "${search_dir}" -type f -regex ".*core\.[0-9]+"
+
+  user_decision "Do you want to delete those files?" user_answer
+  if [[ " ${positive_answers[*]} " =~ " ${user_answer} " ]]; then
+  find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```
Comment (destogl) [inline scripts/_Team_Defines.bash:102]:
```suggestion
    find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```

File: scripts/_Team_Defines.bash [hunk 6]
```diff
@@ -81,6 +81,31 @@ alias ros2_control_setup-controller-package=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/ro
...
+    search_dir="${HOME}"
+  else
+    search_dir=$1
+  fi
+
+  find "${search_dir}" -type f -regex ".*core\.[0-9]+"
+
+  user_decision "Do you want to delete those files?" user_answer
+  if [[ " ${positive_answers[*]} " =~ " ${user_answer} " ]]; then
+    find "${search_dir}" -type f -regex ".*core\.[0-9]+"
```
Comment (destogl) [inline scripts/_Team_Defines.bash:102]:
```suggestion
    find "${search_dir}" -type f -regex ".*core\.[0-9]+" -delete
```

**Pull request merged**